src/clj_meowallet/http.clj

```diff
@@ -30,37 +32,68 @@
     (json/generate-string body)
     body))
 
+(defn get-status-from-error
+  [response]
+  (-> (.getMessage response)
+      (str/split #" ")
```

Comment:
well actually I tried with some different responses and it worked. If I don't do something like this, my :status will be something like "status 4xx". I really would like to respond with {:status 4xx} instead of {:status "status 4xx"}.
what do you purpose?